src/layouts/UseCases.tsx

```diff
@@ -38,12 +38,11 @@ const HeroContainer = (props: ChildOnlyProp) => (
   <Flex
     bg="cardGradient"
     boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
-    direction={{ base: "column-reverse", lg: "row" }}
-    justify="end"
-    minHeight={{ base: "800px", lg: "608px" }}
+    flexDirection={{ base: "column-reverse", lg: "row" }}
+    justifyContent="flex-end"
+    minHeight="608px"
```

Comment:
Just curious if we have a preference for full style prop names vs shorthands... we seem to primarily use shorthand forms, ie `m` over `margin`, and see a lot of `direction` on Chakra `<Flex>` boxes instead of `flexDirection`, though others such as `position` which can be shortened to `pos` we tend to not use... @pettinarip  Was there a reason we flipped these in particular? Should we document our preference here to try and standardize?

Reply:
No strong opinion or preference about this. I think most of the time I use the shorthand version but wouldn't mind changing to a more strict convention. In this case, I didn't notice tbh, I think I was just adjusting things and used what the autocomplete suggested me xD

I wouldn't document these type of things since it doesn't add too much value IMO and makes the code difficult to maintain (you will need to review all these things in each code review, unless you have an automation to do that).